src/main/java/com/blackducksoftware/integration/hub/alert/channel/manager/DistributionChannelManager.java

```diff
@@ -69,8 +69,8 @@ public ObjectTransformer getObjectTransformer() {
         return objectTransformer;
     }
 
-    public ProjectData getTestMessageProjectData() {
-        return new ProjectData(DigestTypeEnum.REAL_TIME, "Hub Alert", "Test Message", Collections.emptyList(), Collections.emptyMap());
+    public String testGlobalConfig(final G globalConfigEntity) {
+        return getDistributionChannel().testGlobalConfig(globalConfigEntity);
```

Comment:
Create a new event here to pass along with our entity

Reply:
It's a test, so there shouldn't be an event. Especially because it's a global config.